doc/reference/aliases/doc.rst

```diff
@@ -0,0 +1,4 @@
+@doc
+====
+
+This alias builds documentation for public libraries as HTML pages.
```

Comment:
I think it would be useful to link to the "Generating documentation" document here.